install_config/configuring_openstack.adoc

```diff
@@ -120,20 +128,19 @@ kubernetesMasterConfig:
     cloud-provider:
       - "openstack"
     cloud-config:
-      - "/etc/cloud.conf"
+      - "/etc/origin/cloudprovider/openstack.conf"
   controllerArguments:
     cloud-provider:
       - "openstack"
     cloud-config:
-      - "/etc/cloud.conf"
+      - "/etc/origin/cloudprovider/openstack.conf"
 ----
 
 [IMPORTANT]
```

Comment:
IMO we don't need this note for 3.10 and 3.11. Master service is only provided via static pod, no rpm-install or containerized install concept for master any more